tests/factories/match_tracker.py

```diff
@@ -26,3 +27,24 @@ class MatchResultFactory(factory.Factory):
 
     class Meta:
         model = dataclasses.MatchResult
+
+
+def build_match_history_between(
+    user_one: core_dataclasses.User, user_two: core_dataclasses.User, games_played: int = 10
+) -> dataclasses.Matches:
+    """
+    Build a realistic match history between two users, where the winner is randomly chosen.
+    """
+    results = []
+    for _ in range(games_played):
+        winner_score = 11
```

Comment:
There is a possibility that a score is an _overtime_ score. You could do a random check so like 10% of tests are an 'overtime' score, then do a winner_score of >11, then the loser_score would just be winner_score - 2.